parsers/EIA.py

```diff
@@ -501,52 +523,41 @@ def _fetch_series(
                 f"target_datetime must be a valid datetime - received {target_datetime}"
             )
         utc = tz.gettz("UTC")
-        if isinstance(target_datetime, datetime):
-            # eia currently only accepts utc timestamps in the form YYYYMMDDTHHZ
-            end = target_datetime.astimezone(utc).strftime("%Y%m%dT%HZ")
-            start = (target_datetime.astimezone(utc) - timedelta(days=1)).strftime(
-                "%Y%m%dT%HZ"
-            )
-        else:
-            end = None
-            start = None
-        raw_data = series.get_data(start=start, end=end)
+        eia_ts_format = "%Y-%m-%dT%HH"
+        end = target_datetime.astimezone(utc)
```

Comment:
In order to convert the Electricity Maps datetime format to EIA api I think we need to add 1 hour here.
```suggestion
        end = target_datetime.astimezone(utc) + timedelta(hours=1)
```